src/feature-selector-form/feature-selector-form.tsx

```diff
@@ -94,7 +94,7 @@ export function FeatureSelectorForm( { onContinue }: Props ) {
 			<div className={ styles.searchWrapper }>
 				<DebouncedSearch
 					callback={ handleSearch }
-					placeholder="Search for a feature"
+					placeholder='Search for a feature (e.g. "Site editor") '
```

Comment:
Great choice in placeholder text! 👍 
 
**super nitpick (non-blocking):** I feel like the `S` in `"Site editor"` should be lowercase to match the sentence case of the placeholder text. That was my first impression, at least. I'm not an expert on this kind of stuff though 😅